delira/models/classification/ResNet18.py

```diff
@@ -6,68 +6,133 @@
 relu = tf.keras.layers.ReLU
 gap2d = tf.keras.layers.GlobalAveragePooling2D
 batchnorm2d = tf.keras.layers.BatchNormalization
+add = tf.keras.layers.Add
+
+
+def get_image_format_and_axis():
+    image_format = tf.keras.backend.image_data_format()
+    if image_format == "channels_first":
+        return image_format, 1
+    elif image_format == "channels_last":
+        return image_format, -1
+    else:
+        raise Exception("Image format type unknown")
```

Comment:
Please make this either RuntimeError or ValueError... Using bare Exceptions is not recommended, since we cannot explicitly catch this type of exception (if we'd like to) and ignore all the others.

Reply:
adressed in 5b55552af1b8b30dcb7c7af257bb34270cc6c5bb